### ISSUE TITLE
Get AD roles for Windows user

### ISSUE THREAD

**Issue by arc95** [MohammadYounes/OWIN-MixedAuth#65]
Is there a way to fetch the user's AD roles after they're authenticated on Windows and limit the app to a few AD groups. Thanks.

**Comment by MohammadYounes** [comment]
See #38. Thanks.

**Comment by MohammadYounes** [comment]
A complete example can be found here https://github.com/MohammadYounes/OWIN-MixedAuth/issues/9#issuecomment-82575987

**Comment by arc95** [comment]
@MohammadYounes! Thanks. Where do call that `foreach`?

**Comment by MohammadYounes** [comment]
Inside the provider's [`OnImportClaims`](https://github.com/MohammadYounes/OWIN-MixedAuth/blob/master/src/MVC/App_Start/Startup.Auth.cs#L82) method.

**Comment by arc95** [comment]
@MohammadYounes, thanks. When I tried using that code in ConfigureAuth, Visual Studio says: 

- 'IAppBuilder' does not contain a definition for 'UseMixedAuth' and no extension method 'UseMixedAuth' accepting a first argument of type 'IAppBuilder' could be found (are you missing a using directive or an assembly reference?
- The type or namespace name 'MixedAuthOptions' could not be found (are you missing a using directive or an assembly reference?
- The type or namespace name 'MixedAuthProvider' could not be found (are you missing a using directive or an assembly reference?
- The name 'cookieOptions' does not exist in the current context	MixedAuth

**Update:** I think I found a sample for that: https://github.com/MohammadYounes/OWIN-MixedAuth/blob/master/src/MVC/App_Start/Startup.Auth.cs#L73
